### ISSUE TITLE
Skills Project: Quickblade Update

### ISSUE THREAD

**Issue by BoldestDungeon** [BoldestDungeon/wildermyth-drauven-pcs#320]
## What would you like to see added?

* Update Quickblade so that it gives the character a free movement action at the end of the turn if the character makes a melee attack, rather than give an additional melee strike.

This will make the Quickblade ability act more like how Haunts behave on the battlefield, allowing the player to reposition themselves after landing a blow.

Corresponding updates to the upgrade ability to be tackled in a follow-up task once we identify a good upgrade bonus.

**Comment by BipenTheBat** [comment]
Ability has gotten the update just waiting on MR approval closing issue
